wemake_python_styleguide/logic/nodes.py

```diff
@@ -29,3 +32,36 @@ def get_parent(node: ast.AST) -> Optional[ast.AST]:
 def get_context(node: ast.AST) -> Optional[ContextNodes]:
     """Returns the context or ``None`` if node has no context."""
     return getattr(node, 'wps_context', None)
+
+
+def does_loop_contain_node(  # TODO: move, reuse in annotations.py
+    loop: Optional[_AnyLoop],
+    to_check: ast.Break,
+) -> bool:
+    """Helper function for has_break method."""
```

Comment:
Please, change the description to something more meaningful.

Reply:
Done!